mis_builder/models/mis_report.py

```diff
@@ -797,6 +797,17 @@ def prepare_aep(self, company):
         aep.done_parsing()
         return aep
 
+    def prepare_localsdict(self):
```

Comment:
@ThomasBinsfeld Oh, one last small thing: can you rename to `prepare_locals_dict` (with the underscore). Since we work a lot by grepping it's always better to be very consistent when naming things.